### ISSUE TITLE
backup_from_standby test fails

### ISSUE THREAD

**Issue by bwtakacy** [ossc-db/pg_rman#10]
I found the master and REL9_5_STABLE branches fail in the backup_from_standby test from this morning.

The standby server did not start and show the below messages
```
LOG:  database system was interrupted; last known up at 2015-09-30 14:11:10 JST
FATAL:  the database system is starting up
FATAL:  the database system is starting up
FATAL:  the database system is starting up
LOG:  entering standby mode
LOG:  restored log file "000000010000000000000004" from archive
LOG:  redo starts at 0/4000028
LOG:  consistent recovery state reached at 0/40000F8
LOG:  database system is ready to accept read only connections
cp: cannot stat `/home/postgres/ossc-db/pg_rman/results/backup_from_standby/arclog/000000010000000000000005': No such file or directory
LOG:  started streaming WAL from primary at 0/5000000 on timeline 1
LOG:  file "pg_commit_ts/0000" doesn't exist, reading as zeroes
CONTEXT:  xlog redo Transaction/COMMIT: 2015-09-30 14:11:16.340063+09; rels: pg_tblspc/16384/PG_9.5_201508061/16385/16386; inval msgs: catcache 45 catcache 44 relcache 16386
LOG:  file "pg_commit_ts/0000" doesn't exist, reading as zeroes
CONTEXT:  xlog redo Transaction/COMMIT: 2015-09-30 14:11:21.916812+09
FATAL:  could not access status of transaction 3276
DETAIL:  Could not read from file "pg_commit_ts/0000" at offset 32768: Success.
CONTEXT:  xlog redo Transaction/COMMIT: 2015-09-30 14:11:33.443691+09
LOG:  startup process (PID 8263) exited with exit code 1
LOG:  terminating any other active server processes
```

Seems to be related to commit timestamp feature.

**Comment by bwtakacy** [comment]
I also have found this commit breaks pg_rman test
```
commit d8c7bb21ea2a3122cac96b2996d3122f25c160c3
Author: Alvaro Herrera <alvherre@alvh.no-ip.org>
Date:   Tue Sep 29 14:40:56 2015 -0300

    Code review for transaction commit timestamps

    There are three main changes here:

    1. No longer cause a start failure in a standby if the feature is
    disabled in postgresql.conf but enabled in the master.  This reverts one
    part of commit 4f3924d9cd43; what we keep is the ability of the standby
    to activate/deactivate the module (which includes creating and removing
    segments as appropriate) during replay of such actions in the master.

    2. Replay WAL records affecting commitTS even if the feature is
    disabled.  This means the standby will always have the same state as the
    master after replay.

    3. Have COMMIT PREPARE record the transaction commit time as well.  We
    were previously only applying it in the normal transaction commit path.

    Author: Petr Jelinek
    Discussion: http://www.postgresql.org/message-id/CAHGQGwHereDzzzmfxEBYcVQu3oZv6vZc
    Discussion: http://www.postgresql.org/message-id/CAHGQGwFuzfO4JscM9LCAmCDCxp_MfLvN

    Additionally, I cleaned up nearby code related to replication origins,
    which I found a bit hard to follow, and fixed a couple of typos.

    Backpatch to 9.5, where this code was introduced.

    Per bug reports from Fujii Masao and subsequent discussion.
```

**Comment by bwtakacy** [comment]
I have found that this can be reproduce without pg_rman, so this seems to be caused by PostgreSQL server side.

I will report to pgsql-hackers.

**Comment by ooyamams** [comment]
Thanks @bwtakacy !
I will check pgsql-hackers too.

**Comment by bwtakacy** [comment]
This has been fixed by https://github.com/postgres/postgres/commit/f12e814b88d8082804bbc8b827469d8068e7252c.

And I have confirmed that all the regression tests are passed in Travis CI.
